src/app/CMakeLists.txt

```diff
@@ -230,7 +230,7 @@ endif()
 set(_install_targets ddApp directorPython drake-visualizer)
 
 add_executable(drake-visualizer drakeVisualizerApp.cpp)
-target_link_libraries(drake-visualizer ddApp)
+target_link_libraries(drake-visualizer ddApp ${PYTHON_LIBRARIES})
```

Comment:
It is actually not necessary. Removing it. (I may have had to add it at the beginning because of another issue due to VTK, not 100% sure)